src/esp/gfx/Drawable.h

```diff
@@ -18,6 +18,15 @@ namespace gfx {
 
 class DrawableGroup;
 
+enum class DrawableType : uint8_t {
+  None = 0,
+  Generic = 1,
+  Pbr = 2,
+  PTexMesh = 3,
+  MeshVisualizer = 4,
+  VarianceShadowMap = 6,
```

Comment:
Was 5 intentionally skipped here? :)

Reply:
Haha, you got sharp eyes. I used to have a `DepthMap` drawable here. I am still not so sure if I would introduce it to the sytem or not. :)